### ISSUE TITLE
Remove --filesystem=home

### ISSUE THREAD

**Issue by nanonyme** [flathub/org.gnome.Eolie#20]
This gives the application write access to home directory and allows executing arbitrary code outside sandbox through injection to eg .profile or other files. If the app really needs access to home, it should be read-only. Why does it need home access in the first place?

**Comment by bellegarde-c** [comment]
@nanonyme It's needed for bookmarks import, readonly should be ok. Or better, Firefox/Chromium/Chrome paths.

**Comment by nanonyme** [comment]
Yes, definitely exact paths to those is fine. It also makes intent more obvious to user.
